src/__tests__/allComponenets.test.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
+import Home from '../pages/Home';
+import CountryData from '../components/CountryData';
+import Navbar from '../components/Navbar';
+import InputFilter from '../components/InputFilter';
+import RegionsData from '../components/RegionsData';
+
+describe('All components/pages render correctly', () => {
```

Comment:
- [ ] Kindly write integration tests using the `React Testing Library` as stated in the requirements.